src/main/java/com/google/devtools/build/lib/remote/common/FutureCachedActionResult.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2014 The Bazel Authors. All rights reserved.
```

Comment:
Please move this to `RemoteCacheClient` just like `ActionKey`.

Reply:
done, auto value on a nested class took a little searching :(